### PR TITLE
Feature/improve excel report

### PR DESCRIPTION
**Pull request by lewisdaly**
This PR makes a number of improvements to the license-scanner, namely:

- Auto generate summaries in the merged xlsx document in standalone mode
- tidying and clarifying different modes (docker, standalone, local)
- More explicity defines disallowed licenses to the fail-list in `config.toml`
- Change the `license-scanner` dependency to a custom git location as a temporary measure, as we wait for this pr to be merged and released: https://github.com/davglass/license-checker/pull/211

This has been tested on `central-ledger` with a throwaway PR.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -40,34 +41,77 @@ pathToRepo = ""
...
 
 ##
 # The list of repos to scan
-
 # Lines starting with a # will be ignored
 # Ignored when `mode != "standalone"`
 ##
-repos = [ ]
-
+repos = [
```
Comment (elnyry-sam-k) [inline config.toml:51]: I guess we don't need central-end-user-registry, central-hub..

File: lib/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "license_scanner",
-  "version": "1.0.0",
+  "version": "7.2.0",
```
Comment (elnyry-sam-k) [inline lib/package.json]: 7.3.0

File: config.toml (hunk 1 above)
Comment (lewisdaly) [reply config.toml:51]: Cool cool.

File: lib/package.json (hunk 2 above)
Comment (lewisdaly) [reply lib/package.json]: Hmm good point.

**Review by kjw000 — APPROVED** [review]
+1

**Review by rmothilal — APPROVED** [review]
Happy

**Pull request merged**